### ISSUE TITLE
Tutorial step Getting Dynamic: Error: Cannot call write after a stream was destroyed

### ISSUE THREAD

**Issue by hesselberg** [redwoodjs/redwood#405]
Possibly similar to #404 

On step Getting Dynamic, and applying the migration via `yarn rw db up`, the generate Prisma client fails.

Here's the log:

```
➜  redwoodblog git:(master) ✗ yarn rw db up
yarn run v1.22.4
$ './redwoodblog/node_modules/.bin/rw' db up
Migrate database up... [started]
$ './redwoodblog/node_modules/.bin/prisma2' migrate up --experimental --create-db
🏋️‍  migrate up

All migrations are already applied
Migrate database up... [completed]
Generating the Prisma client... [started]
$ './redwoodblog/node_modules/.bin/prisma2' generate
Error: Cannot call write after a stream was destroyed
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Generating the Prisma client... [failed]
→ Command failed with exit code 1: yarn prisma2 generate
Command failed with exit code 1: yarn prisma2 generate
✨  Done in 2.55s.
➜  redwoodblog git:(master) ✗

```

**Comment by thedavidprice** [comment]
Hi @hesselberg sorry for the problems here and hope we can get you up and running shortly!

Note: this is most likely a _different_ issue than #404 

It's possible something is out of step between:
1) updating `schema.prisma` 
2) running `yarn rw db save` and 
3) running `yarn rw db up`

If you haven't already, could you delete:
- prisma/migrations directory
- prisma/dev.db file

Then re-run `yarn rw db save` followed by `yarn rw db up`

If you'd like to compare your code to a final version of the tutorial, use [this repository](https://github.com/thedavidprice/redwood-tutorial-test) as a reference.

Any luck after trying the steps above?

**Comment by thedavidprice** [comment]
closed by #457
